torch/onnx/symbolic_opset9.py

```diff
@@ -1084,10 +1084,11 @@ def __interpolate(g, input, size, scale_factor, mode , align_corners, recompute_
                                                              mode , align_corners)
     return g.op("Upsample", input, scales, mode_s=mode)
 
-@parse_args("v")
+
 def bitwise_not(g, inp):
     if inp.type().scalarType() != "Bool":
-        return _unimplemented("bitwise_not", "non-bool tensor")
+        raise NotImplementedError("ONNX export does NOT support exporting bitwise Not " +
+                                  "for non-boolean input values")
```

Comment:
String literals are automatically concatenated https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation
```suggestion
        raise NotImplementedError("ONNX export does NOT support exporting bitwise Not "
                                  "for non-boolean input values")
```